packages/components/src/components/editor/condition-builder/conditionBuilder.css.ts

```diff
@@ -1,8 +1,24 @@
 import { vars } from '@/styles/theme.css';
 import { style } from '@vanilla-extract/css';
 
-export const conditionBorder = style({
+const sharedConditionStyles = {
   border: vars.border.basic,
   borderRadius: vars.border.r2,
   padding: vars.size.s2
+};
+
+export const conditionBorder = style(sharedConditionStyles);
+
+export const conditionGroupBorder = style({
+  selectors: {
+    '&[data-condition-mode="nested-condition"]': sharedConditionStyles
+  }
+});
+
+export const operatorField = style({
+  width: '150px'
```

Comment:
just as info, if you have px then you can also write the following if you like:
```suggestion
  width: 150
```